features/steps/app_steps.rb

```diff
@@ -9,6 +9,9 @@ def launch_crashy_app(os, event)
   run_required_commands([["features/scripts/launch_#{os}_app_crash.sh"]])
 end
 
+When(/^I launch an (\w+) app with "(\w+)"$/) do |os, event_type|
```

Comment:
I would be nice to convert the previous test cases to use this generic step…

a) for consistency
b) so that not every new test has to write a new step
c) the steps below can all be removed

It could also have a more descriptive title than "with", e.g.:

`"When I run the {string} scenario on a(n?) {string} device"`